src/lang-promql/complete/promql.terms.ts

```diff
@@ -467,4 +467,25 @@ export const snippets: readonly Completion[] = [
     info: 'Set or replace a label value in an input vector',
     apply: snippet('label_replace(${__input_vector__}, "${__dst__}", "${__replacement__}", "${__src__}", "${__regex__}")'),
   },
+  {
+    label: 'topk(__top_rank_number__, __metric__)',
+    type: 'function',
+    detail: 'snippet',
+    info: 'Largest k elements by sample value',
+    apply: snippet('topk(${__top_rank_number__}, ${__metric__})'),
+  },
+  {
+    label: 'bottomk(__bottom_rank_number__, __metric__)',
+    type: 'function',
+    detail: 'snippet',
+    info: 'Smallest k elements by sample value',
+    apply: snippet('bottomk(${__bottom_rank_number__}, ${__metric__})'),
+  },
+  {
+    label: 'count_values("__label_name__", __metric__)',
+    type: 'function',
+    detail: 'snippet',
+    info: 'Count number of elements with the same value',
```

Comment:
To make it even clearer:

```suggestion
    info: 'Count the number of series per distinct sample value',
```